src/System.Json/src/System/Json/JsonPrimitive.cs

```diff
@@ -159,8 +159,8 @@ internal string GetFormattedString()
 
                 case JsonType.Number:
                     string s = _value is float || _value is double ?
-                        ((IFormattable)_value).ToString("R", NumberFormatInfo.InvariantInfo) : // Use "round-trip" format
-                        ((IFormattable)_value).ToString("G", NumberFormatInfo.InvariantInfo);
+                        ((IFormattable)_value).ToString("R", CultureInfo.InvariantCulture) : // Use "round-trip" format
```

Comment:
Is this considered a breaking change (esp if this came from mono)?

Reply:
Technically, yes, but then again, isn't this generating invalid JSON as it was before?  That would seem to override any other breaking change concerns.  For example, if your culture is fr-FR, this will generate floating-point values, TimeSpans, etc. that use a comma instead of a period for the decimal separator.